docs/Developers/Newcomers_Guide.rst

```diff
@@ -97,113 +109,114 @@ Step 2. Grab an Invitation to the Organization
 
 Now that you are part of our organization, you can start working on issues.
 If you are familiar with git, you can skip the next section and pick an issue.
 
 Optional. Get Help With Git
 ---------------------------
 
-We use GitHub to manage our repository. If you're not familiar with git, we
-strongly recommend following a tutorial, such as `this one <https://try.github.io/levels/1/challenges/1>`_.
+We use GitHub to manage our repository.
+If you're not familiar with git, we strongly recommend following a tutorial,
+such as `this one <https://try.github.io/levels/1/challenges/1>`_.
 
 We also have a page dedicated to git commands that will help you learn the
-basics: `here <http://coala.io/git>`_.
-
-If there's anything unclear, or you are encountering problems, feel free
-to contact us on `gitter <https://coala.io/chat>`_,
-and we will help you!
+basics: `here <http://coala.io/git>`__.
 
 Step 3. Picking Up an Issue
 ---------------------------
 
 Now it is time to pick an issue.
 It is the best way to familiarise yourself with the codebase.
-Here is the link that will lead you to `Newcomers issues <https://coala.io/new>`_.
-
-.. note::
-
-    You need to be logged in before you follow the Newcomers issues link.
-
-.. seealso::
-
-    For more information about what bears are, please check the following link: `Writing Native bears <http://api.coala.io/en/latest/Developers/Writing_Native_Bears.html>`_
+Here is the link that will lead you to
+`newcomer issues <https://coala.io/new>`_.
+Note however, that you need to be logged in before you can follow the Newcomers
+issues link.
 
 The easy issues that will help you get started are labeled as
 ``difficulty/newcomer`` and are only there to give you a glimpse of how it is
-to work with us and regarding the workflow.
+to work with us and learning the workflow.
+Look through the list for an issue that you feel comfortable to work on and
+understand what has to be changed without having to ask for help in the chat.
 
-Now pick an issue which isn't assigned, and if you want to fix
-it, then leave a comment that you would like to get assigned. This way
-we don't have multiple people working on the same issue at the same time.
-Now you can start working on it.
+Now pick an issue which isn't assigned.
+As stated before, you should never work on an issue without being assigned to
+it.
+Fortunately, cobot is here to help you!
+So, if you are interested in picking up an issue just write the following
+command in the gitter chat::
 
-.. note::
-
-    As stated before, you should never work on an issue without any
-    assignment. Fortunately, cobot is here to help you! So, if you are
-    interested in picking up an issue just write in the gitter chat the
-    following command::
+   cobot assign <issue_link>
 
-        cobot assign <issue_link>
 
-    Take care to write the full link to the issue
+Take care to write the full link to the issue.
```

Comment:
`Make sure you use the full link to issue while assigning`, this seems better IMHO.